uv-build is not available on Solaris. This patch reverts to hatchling
which was used by this project before.

https://github.com/rthalley/dnspython/commit/78792389627cf9bbd41022b92f600be38626d355

--- dnspython-2.9.0/pyproject.toml
+++ dnspython-2.9.0/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["uv_build>=0.9.21,<0.13.0"]
-build-backend = "uv_build"
+requires = ["hatchling>=1.21.0"]
+build-backend = "hatchling.build"
 
 [project]
 name = "dnspython"
@@ -55,10 +55,8 @@ repository = "https://github.com/rthalle
 documentation = "https://dnspython.readthedocs.io/en/stable/"
 issues = "https://github.com/rthalley/dnspython/issues"
 
-[tool.uv.build-backend]
-module-name = "dns"
-module-root = ""
-source-include = [
+[tool.hatch.build.targets.sdist]
+include = [
     "dns/*.py",
     "dns/**/*.py",
     "dns/py.typed",
@@ -78,11 +76,14 @@ source-include = [
     "tests/tsigkeys/*.key",
     "util/**",
 ]
-source-exclude = [
+exclude = [
     ".pytest_cache",
     ".DS_Store",
 ]
 
+[tool.hatch.build.targets.wheel]
+include = ["dns/*.py", "dns/**/*.py", "dns/py.typed"]
+
 [tool.ruff]
 lint.select = [
     # pycodestyle
